packages/api/src/command/medical/document/document-query.ts

```diff
@@ -81,6 +82,11 @@ export async function queryDocumentsAcrossHIEs({
     requestId,
   }).catch(emptyFunction);
 
+  getDocumentsFromCQ({
+    patient,
+    requestId,
+  }).catch(emptyFunction);
```

Comment:
For things like `forceDownload` and `forceQuery` not handling in this PR since doc retrieval will be worked on in a separate PR

Reply:
Also the question for V1 is if we want to include this func or add it after the fact?

Reply:
We have the concept of `forceDownload`. With the definition being Whether to override files already downloaded (optional, defaults to false).
- What this does is it re downloads the docs even if they are on s3
- My assumption as to why we would do this is because we may have docs that didn't download correctly so we override to download and overwrite them. 
- We also have an endpoint called populate-fhir-server which I don't think we use anymore

Reply:
We also have the concept of `forceQuery`. 
- It resets the docQueryProgress
- And starts the query process again 
- We have an internal endpoint that uses forceQuery
- Assuming if something is in a borked state then we want to reset it